scripts/py_matter_yamltests/matter_yamltests/definitions.py

```diff
@@ -87,6 +87,9 @@ def __init__(self, sources: List[ParseSource]):
                     self.__responses_by_id[code][struct.code] = struct
                     self.__responses_by_name[name][struct.name] = struct.code
 
+    def get_cluster_names(self) -> List[str]:
```

Comment:
This would be more generalized as "get_cluster_names_by_ids" that returns `Dict[int, str]`. The names is `list(obj.get_cluster_names_by_ids().values())`, for example. Could be done as a separate method.

Reply:
Well, I want to match the name of the other methods:
 1. `get_command_names`
 2. `get_attributes_names`
 3. `get_event_names`